spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -160,44 +160,44 @@ def resolve(path, http_verb=:get)
 
   end
 
-  context 'hash_for helpers' do
-
-    it 'should return a localized route when using hash_for helpers passing in locale' do
-      I18n.locale = 'fr'
-      route_hash = hash_for_posts_url(:locale => 'es')
-      route_hash.should eql :locale=>'es', :action=>"index", :controller=>"posts", :use_route=>"posts", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos"
-
-      route_hash = hash_for_posts_url(:locale => :es)
-      route_hash.should eql :locale=>:es, :action=>"index", :controller=>"posts", :use_route=>"posts", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos"
-    end
-
-    it 'should respect I18n.locale' do
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash.should eql :id=>1, :action=>"show", :controller=>"posts", :use_route=>"post", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/posts/1"
-
-      I18n.locale = 'es'
-
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash.should eql :id=>1, :action=>"show", :controller=>"posts", :use_route=>"post", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos/1"
-    end
-
-  end
-
-  context 'url_for' do
-
-    it 'should recognize non named routes' do
-      I18n.locale = 'es'
-
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash = route_hash.slice!(:use_route)
-      route_hash.should eql :id => 1, :action => "show", :controller => "posts", :only_path => false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos/1"
-
-    end
-  end
+  # context 'hash_for helpers' do
```

Comment:
The `hash_for` URL helper methods were removed in Rails 4.2. See [3.2](https://github.com/rails/rails/blob/3-2-stable/actionpack/lib/action_dispatch/routing/route_set.rb) vs [4.2](https://github.com/rails/rails/blob/4-2-stable/actionpack/lib/action_dispatch/routing/route_set.rb) where the `define_hash_access` method isn't in 4.2. Does this mean these tests won't be necessary anymore? I couldn't see a way of refactoring these tests to make them applicable.